app/models/search/pseud_query.rb

```diff
@@ -28,12 +28,29 @@ def collection_filter
     { term: { collection_ids: options[:collection_id] } } if options[:collection_id]
   end
 
+  def fandom_filter
+    if options[:fandom_ids]
+      options[:fandom_ids].map do |fandom_id|
+        { term: { "fandoms.id" => fandom_id } }
+      end
+    end
+  end
+
   ###########
   # QUERIES
   ###########
 
   def general_query
-    { query_string: { query: options[:query] } } if options[:query]
+    {
+      simple_query_string:{
+        query: escape_reserved_characters(options[:query]),
+        fields: ["name^5", "user_login^2", "description"],
```

Comment:
👍 

For my own reference, this means name is 5 times as important as description. https://www.elastic.co/guide/en/elasticsearch/reference/6.0/query-dsl-multi-match-query.html#_literal_fields_literal_and_per_field_boosting